### PR TITLE
Reformat coordinate transforms

### PR DESCRIPTION
**Pull request by MaxHellstrom**
Reformat coordinate transforms and orientation to make the calculation model consistent.

This includes phantom rotation and directions, rotational matrices, patient positioning, etc.

### CODE REVIEW

File: src/pyskindose/geom_calc.py [hunk 1]
```diff
@@ -48,22 +47,19 @@ def calculate_field_size(field_size_mode, data_parsed, data_norm):
...
+def position_patient_phantom_on_table(
         patient: Phantom,
         table: Phantom,
         pad: Phantom,
         pad_thickness: Any,
         patient_offset: List[int],
         patient_orientation: c.PATIENT_ORIENTATION_HEAD_FIRST_SUPINE
                     ) -> None:
-    """Manual positioning of the phantoms before procedure starts.
+    """Manual positioning of the phantom upon the patient support table.
```
Comment (BwKodex) [inline src/pyskindose/geom_calc.py:58]: Is it correct to call it a _manual positioning_?

File: src/pyskindose/phantom_class.py [hunk 2]
```diff
@@ -250,22 +253,21 @@ def __init__(self,
...
-            #           0.9, 0.9, 1, 1, 0.9, 0.9, 0, 0]]
+            y_tab = [index * phantom_dim.table_thickness for index in
+                     [0, 0, 0, 0, 0, 0, 0, 0, +1, +1, +1, +1, +1, +1, +1, +1]]
+
+            # Vertical position of the vertices
+            z_tab = [index * phantom_dim.table_length for index in
+                     [0, 0, 0, 0, 0, 0, -1, -1, 0, 0, 0, 0, 0, 0, -1, -1]]
+
+            # Lateral position of the vertices. Replace the list z-list with
+            #z_tab = [index * phantom_dim.table_length for index in
```
Comment (BwKodex) [inline src/pyskindose/phantom_class.py:264]: Remove commented out code

File: src/pyskindose/phantom_class.py [hunk 3]
```diff
@@ -289,14 +291,20 @@ def __init__(self,
...
 
             # Vertical position of the vertices
-            z_pad = [index * phantom_dim.pad_thickness for index in
-                     [0, 0, 0, 0, 0, 0, 0, 0,
-                      1, 1, 1, 1, 1, 1, 1, 1]]
+            z_pad = [index * phantom_dim.pad_length for index in
+                     [0, 0, 0, 0, 0, 0, -1, -1, 0, 0, 0, 0, 0, 0, -1, -1]]
+
+            # Lateral position of the vertices. Replace the list z list with
+            # z_pad = [index * phantom_dim.table_length for index in
```
Comment (BwKodex) [inline src/pyskindose/phantom_class.py:302]: Remove commented out code

File: src/pyskindose/phantom_class.py [hunk 4]
```diff
@@ -289,14 +291,20 @@ def __init__(self,
...
+            y_pad = [index * phantom_dim.pad_thickness for index in
+                     [0, 0, 0, 0, 0, 0, 0, 0, -1, -1, -1, -1, -1, -1, -1, -1]]
 
             # Vertical position of the vertices
-            z_pad = [index * phantom_dim.pad_thickness for index in
-                     [0, 0, 0, 0, 0, 0, 0, 0,
-                      1, 1, 1, 1, 1, 1, 1, 1]]
+            z_pad = [index * phantom_dim.pad_length for index in
+                     [0, 0, 0, 0, 0, 0, -1, -1, 0, 0, 0, 0, 0, 0, -1, -1]]
+
```
Comment (BwKodex) [inline src/pyskindose/phantom_class.py:300]: This seems to be kind of a duplicate of what's happening at row 256-261. Could we break it out to not have the same code repeating?

File: src/pyskindose/geom_calc.py (hunk 1 above)
Comment (MaxHellstrom) [reply src/pyskindose/geom_calc.py]: Nope. I have updated this sentence in b9fe4e9d922ae722ddaf41a93f0b0450a67aa60a

File: src/pyskindose/phantom_class.py (hunk 2 above)
Comment (MaxHellstrom) [reply src/pyskindose/phantom_class.py]: done 1283f6aa1ddd67c3336a8523efb4f97a53124de8

File: src/pyskindose/phantom_class.py (hunk 3 above)
Comment (MaxHellstrom) [reply src/pyskindose/phantom_class.py]: done 1283f6aa1ddd67c3336a8523efb4f97a53124de8

File: src/pyskindose/phantom_class.py (hunk 4 above)
Comment (MaxHellstrom) [reply src/pyskindose/phantom_class.py]: This has been improved in 2e0c900c95f5c1c11a2e4f67cb162cdd8f585edc

File: src/pyskindose/geom_calc.py [hunk 5]
```diff
@@ -417,13 +407,13 @@ def check_table_hits(source: np.array, table: Phantom, beam,
     # Create triangles:
 
     # Define edges of table (see illustration in project documentation)
-    a = table.r[6, :]
-    a1 = table.r[7, :]
-    a2 = table.r[5, :]
+    a = table.r[0 + 4, :]
```
Comment (BwKodex) [inline src/pyskindose/geom_calc.py:410]: Is the change in indices for this and row 414 below intentional

File: tests/unittests/test_positioning.py [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+from pyskindose.settings_pyskindose import PyskindoseSettings
+from manual_tests.base_dev_settings import DEVELOPMENT_PARAMETERS
+from pyskindose.geom_calc import position_patient_phantom_on_table
+
+param = PyskindoseSettings(DEVELOPMENT_PARAMETERS)
+
+
+def test_mathematical_phantom_positioning_in_z_direction():
+    # the patient phantom origin is located at the top its head. Therefore,
```
Comment (BwKodex) [inline tests/unittests/test_positioning.py:21]: Check the spelling/missing words in this sentence

File: src/pyskindose/geom_calc.py (hunk 5 above)
Comment (MaxHellstrom) [reply src/pyskindose/geom_calc.py:410]: yes, it was changed to fit the new coordinate standard in e2725669128fd74dc0cb86a1ca14b1bccbc7d658

File: tests/unittests/test_positioning.py (hunk 6 above)
Comment (MaxHellstrom) [reply tests/unittests/test_positioning.py]: fixed in 33a31c900360673925c27eca67cea792c4bd22f4

**Review by BwKodex — APPROVED** [review]

**Pull request merged**